src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -7,13 +7,19 @@ import Link, { LinkProps } from '../Link';
 export interface BreadcrumbItem {
   /** The URL that this Breadcrumbs should navigate to when clicked */
   href: string;
-
-  /** The text that should be displayed on this particular Breadcrumb */
-  text: string;
+  /** Title is the text that should be displayed on this particular Breadcrumb
+   * element can be: numbers, strings, elements or an array (or fragment) containing these types.
+   */
+  title: React.ReactNode;
```

Comment:
Since a breadcrumb may have links, dropdowns, etc., I feel that `node` or `children` is better.  I actually think I prefer `children` unless you don't

Reply:
No I like it, but don't you think this is going to overwrite the children prop from react? 
Do you like the `render` term?
It will be like render props without logic!